packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -3,6 +3,8 @@
  */
 import clsx from 'clsx';
 // TODO: use the @wordpress/components one once public
+// eslint-disable-next-line no-restricted-imports
+import { useStoreState } from '@ariakit/react';
```

Comment:
@WordPress/gutenberg-components WDYT - should we re-export this from `@wordpress/components`?

Reply:
I guess it depends on what we decide around exposing stores in https://github.com/WordPress/gutenberg/issues/63704. Better hold off for now